src/Microsoft.Data.Analysis/PrimitiveDataFrameColumnComputations.tt

```diff
@@ -112,7 +166,61 @@ namespace Microsoft.Data.Analysis
 <# } #>
             }
 <# } else if ((method.MethodName == "CumulativeMax" || method.MethodName == "CumulativeMin" || method.MethodName == "CumulativeProduct" || method.MethodName == "CumulativeSum") && method.SupportsRowSubsets == true) { #>
-            var ret = default(<#=type.TypeName#>);
+<# if (method.MethodName == "CumulativeMax") { #>
+    <# if (type.TypeName == "int") { #>
+        var ret = Int32.MinValue;
```

Comment:
These should use the C# keyword names:

- `int`
- `long`
- etc

Check out [https://blog.paranoidcoding.com/2019/04/08/string-vs-String-is-not-about-style.html](https://blog.paranoidcoding.com/2019/04/08/string-vs-String-is-not-about-style.html) for reasoning.